src/NHibernate.Test/TestCase.cs

```diff
@@ -42,17 +38,7 @@ protected TestDialect TestDialect
 		/// <summary>
 		/// To use in in-line test
 		/// </summary>
-		protected bool IsAntlrParser
-		{
-			get
-			{
-				return sessions.Settings.QueryTranslatorFactory is ASTQueryTranslatorFactory;
-			}
-		}
-
-		private ConcurrentBag<ISession> _openedSessions = new ConcurrentBag<ISession>();
-		private ISession _lastOpenedSession;
-		private DebugConnectionProvider connectionProvider;
+		protected bool IsAntlrParser => Sfi.Settings.QueryTranslatorFactory is ASTQueryTranslatorFactory;
```

Comment:
Was thinking it was very likely obsolete, but had not checked. Thanks for the cleanup.